src/balances/entities/balances.entity.ts

```diff
@@ -0,0 +1,6 @@
+import { Balance } from './balance.entity';
+
+export interface Balances {
+  fiatTotal: number;
+  items: [Balance];
```

Comment:
This is an interesting syntax. Is it the same as `Balance[]` or `Array<Balance>`?

Reply:
> This is an interesting syntax. Is it the same as `Balance[]` or `Array<Balance>`?

Yeah good point! I've been writing with `T[]` but I might have missed this one. This seems to be a `tuple` type. 

However with tuple types the size should be known beforehand so this would mean a tuple of size 1 which corresponds to a `Balance` (but it actually outputs a bigger collection 🤔 ). Do you know why this might be the case?

I will change it.

Reply:
Changed it here [183e0a6](https://github.com/5afe/safe-client-gateway-nest/pull/29/commits/183e0a6448be4b0967d87b95c01e9dde0445f6ee)

Reply:
> Do you know why this might be the case?

You mean why it's not catching that the actual array is bigger than 1? Because the types are static and it cannot see what's actually inserted into this array in runtime unless you're literally pushing things one by one.

Reply:
> > Do you know why this might be the case?
> 
> You mean why it's not catching that the actual array is bigger than 1? Because the types are static and it cannot see what's actually inserted into this array in runtime unless you're literally pushing things one by one.

I took another look and it makes more sense now since we are setting the tuple to a `Balance[]` which is compatible. If later we try to access the second index (`1`) then we get this warning `TS2493: Tuple type '[Balance]' of length '1' has no element at index '1'.`